docs/docs/50-user-guide/60-reference-docs/40-expressions.md

```diff
@@ -443,11 +443,52 @@ You can handle `nil` values gracefully in Expr using its
 features.
 :::
 
-### `commitFrom(repoURL, [freightOrigin])`
+### `commitFrom`
 
-The `commitFrom()` function returns a corresponding `GitCommit` object from the
-`Promotion` or `Stage` their `FreightCollection`. It has one required and one
-optional argument:
+The `commitFrom` function signature varies based on where it's being used.
+
+In the context of `DiscoveredArtifacts`, the function signature is:
```

Comment:
This might not be well understood. Its meaning would probably be more clear if we talked about "Freight creation criteria":

```suggestion
In the context of an optional expression evaluated to determine whether criteria
have been met for automatic creation of a `Freight` resource following
`Warehouse`'s artifact discovery process, the function signature is:
```